pack/tdc/tdcp.json

```diff
@@ -90,20 +90,25 @@
         "type_code": "treachery"
     },
     {
-        "back_flavor": "<b>This is placeholder back text until the card is revealed.</b>",
-        "back_text": "<b>This is placeholder back text until the card is revealed.</b>",
+        "back_flavor": "A Southern transplant raised in New York, Lucius has made quite a name for himself writing dreamy poetry alongside the intellectuals, artists, and musicians of Harlem. What his contemporaries don't know is that the hazy underwater city Lucius describes is inspired not by New York, but by the nightmarish dreams of his missing partner, Rudi. Before he disappeared, Rudi spoke of a cabal of fellow dreamers in Arkham who shared these unsettling visions. Now, Lucius has taken on the role of guest lecturer at Miskatonic University as he searches for his lost lover, and perhaps the source of his own horrifying dreams...",
+        "back_text": "<b>Deck Size</b>: 30.\n<b>Deckbuilding Options</b>: Seeker cards ([seeker]) level 0-5, Rogue cards ([rogue]) level 0-2, Neutral cards level 0-5.\n<b>Deckbuilding Requirements</b> (do not count toward deck size): Book of Verse, Dreams of the Flood, 1 random basic weakness.",
         "code": "11004",
         "deck_limit": 1,
         "deck_options": [
             {
-                "faction": ["neutral"],
+                "faction": ["seeker", "neutral"],
                 "level": {
                     "min": 0,
                     "max": 5
+                },
```

Comment:
The JSON formatting here is incorrect, it should be:
```
[
      {
        "faction": ["seeker", "neutral"],
        "level": {
          "min": 0,
          "max": 5
        }
      },
      {
        "faction": ["rogue"],
        "level": {
          "min": 0,
          "max": 2
        }
      }
    ]
```

Reply:
Oops, you're absolutely right!